weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/tabularListViews/columnBuilder.tsx

```diff
@@ -112,7 +112,7 @@ export function prepareFlattenedDataForTable<T>(
       cleaned[newKey] = flattened[key];
     });
 
-    return cleaned as T & {[key: string]: string};
+    return {...r, ...cleaned} as T & {[key: string]: string};
```

Comment:
this feels like it might be related to the issue, why are we doing this?

Reply:
yeah this is the issue

Reply:
the typing is wrong - two parts of the app make different assumptions

Reply:
commenting this out interestingly just removes the data from the table, but the column persists... there is something else too